pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1138,14 +1138,14 @@ var _ = Describe("ProjectService", func() {
 
 	Describe("healthcheck", func() {
 
-		Context("when valid healthcheck is defined in deploy block", func() {
+		Context("when valid healthcheck and probe type are defined in deploy block", func() {
```

Comment:
Tiny one, the `in deploy block` part can go... I think it was previously included in the context description by mistake.  Same goes for the spec below.